3-UsingAPIs/Week2/homework/ex4-diceRace.js

```diff
@@ -13,12 +13,21 @@ const rollDice = require('../../helpers/pokerDiceRoller');
 function rollTheDices() {
   const dices = [1, 2, 3, 4, 5];
   // TODO complete this function; use Promise.race() and rollDice()
+  const combinedPromise = dices.map((element) => rollDice(element));
+  console.log(combinedPromise);
```

Comment:
Get rid of debug type console.log statements before you make a pull request.

Reply:
Removed